core/tests/protractor_utils/forms.js

```diff
@@ -343,9 +357,10 @@ var AutocompleteMultiDropdownEditor = function(elem) {
       }
 
       for (var i = 0; i < texts.length; i++) {
-        await elem.element(by.css('.select2-container')).click();
-        await elem.element(by.css('.select2-search__field')).sendKeys(
-          texts[i] + '\n');
+        var select2ContainerButton = elem.element(by.css('.select2-container'));
+        await action.click('Test Container Button', select2ContainerButton);
+        var select2SearchField = elem.element(by.css('.select2-search__field'));
+        await action.sendKeys('Field', select2SearchField, texts[i] + '\n');
```

Comment:
Could the name be more descriptive? For example: `Select2 Search Field`

Reply:
updated!